pets.json

```diff
@@ -18,5 +18,15 @@
     "github": "chsienki",
     "instagram": "brotherlucca",
     "twitter": ""
+  },
+  {
+    "name": "Sam",
+    "jobTitle": "Libraries Engineer",
+    "img": "https://github.com/danmoseley/petsof.net/blob/main/sam.jpg",
```

Comment:
I think this should be `https://github.com/danmoseley/petsof.net/raw/main/sam.jpg`  The above link doesn't go to actual file.

Reply:
Yeah, was just about to say the same thing.

Reply:
Or you can add `raw=true` query param to the link you have which should work too. Try opening a file with this in a browser and you'll see the right file gets loaded:
```html
<html>
<img src="https://github.com/danmoseley/petsof.net/blob/main/sam.jpg?raw=true" />
</html>
```

Reply:
Maybe not. Look how the favicon is consumed:

https://github.com/chsienki/petsof.net/blob/cd78dabbb3806a2180925dc3ddf82ccfad7db6d4/template.html#L8

Reply:
It doens't quite work like that, it actually reads the URL and base64 encodes into the HTML file, which is then uploaded to azure where its served from.

Reply:
You expect me to test my changes? 😲